komodo/check_up_to_date_pypi.py

```diff
@@ -147,9 +148,14 @@ def run_check_up_to_date(
         f"{sys.version_info.micro}"
     ),
     propose_upgrade=False,
+    ignore=None,
 ):
     yaml = yaml_parser()
-    releases = load_from_file(yaml, release_file)
+    releases: dict = load_from_file(yaml, release_file)
+    if ignore:
+        for package_name, package_version in list(releases.items()):
+            if re.search(ignore, f"{package_name} {package_version}"):
+                del releases[package_name]
```

Comment:
An alternative way could be to do a dict comprehension, then you could avoid the [list(releases.items())](url) (it is usually an antipattern and some linters could erroneously suggest to remove the `list` function here)

Reply:
Either dict comprehension or dict(filter(filter_function, releases.items()))